docs/core-concepts/core-sdk/cli/dotnet-test.md

```diff
@@ -0,0 +1,99 @@
+dotnet-test
+================
+
+## NAME
+
+`dotnet-test` - runs unit test using the configured test runner
+
+## SYNOPSIS
+
+dotnet-test [--configuration]  
+    [--output] [--build-base-path] [--framework] [--runtime]
+    [--no-build]
+    [--parentProcessId] [--port]  
+    [< project >]  
+
+## DESCRIPTION
+
+`dotnet-test` is used to execute unit tests in a given project. Unit tests are class library 
+projects that have dependencies on the unit test framework (e.g. NUnit or xUnit) and the 
+dotnet test runner for that unit testing framework. These are packaged as NuGet packages and are 
+restored as ordinary dependencies for the project.
+
+Test projects also need to specify a test runner property in project.json using the "testRunner" node. 
+This value should contain the name of the unit test framework.
+
+Below is a sample project.json that shows the needed properties:
```

Comment:
Suggestion: The following project.json sample shows the needed properties *to configure* ...? Also you could list the property names in parentheses perhaps.